metpy/xarray.py

```diff
@@ -78,23 +91,42 @@
 
 @xr.register_dataarray_accessor('metpy')
 class MetPyDataArrayAccessor(object):
-    """Provide custom attributes and methods on XArray DataArray for MetPy functionality."""
+    r"""Provide custom attributes and methods on xarray DataArrays for MetPy functionality.
+
+    This accessor provides several convenient attributes and methds through the `.metpy`
+    attribute on a DataArray. For example, MetPy can identify the coordinate corresponding
+    to a particular axis (given sufficent metadata):
+
+        >>> import xarray as xr
+        >>> temperature = xr.DataArray([[0, 1], [2, 3]], dims=('lat', 'lon'), \
+                                       coords={'lat': [40, 41], 'lon': [-105, -104]}, \
+                                       attrs={'units': 'degC'})
```

Comment:
Looks like the docs are failing on the line break characters...can you wrap the whole thing in () to fix?

Reply:
That's what I had at first, and it wasn't working. It looks like what I actually forgot was the `...` for line continuation

Reply:
Ah, yup that makes sense.